src/sportx.ts

```diff
@@ -612,7 +628,10 @@ class SportX implements ISportX {
       {
         method: "POST",
         body: JSON.stringify(payload),
-        headers: { "Content-Type": "application/json" },
+        headers: {
+          "Content-Type": "application/json",
+          "X-Api-Key": this.apiKey,
```

Comment:
Actually we can append the api key to every request

Reply:
can you elaborate a little more here?